src/web/js-interop.md

```diff
@@ -22,3 +22,77 @@ For help using the `js` package, see the following:
 [js-api]: {{site.pub-api}}/js
 [sass]: {{site.pub-pkg}}/sass
 
+## Future-proof JS interop
+
+Dart's JS interop story is currently evolving. 
+Many of the features that enable future JS interop,
+and support of all three compilers,
+are ready to use (Dart version 2.19). 
+To ease the transition to future JS interop,
+you can refactor your code to conform to the
+syntax and semantics of the new features now.
```

Comment:
I fear this may be too strong as a promise.

The syntax that is available today via `@staticInterop` works semantically similar to the future JSInterop we will have with inline-classes, but both syntax and semantics are expected to change (syntax is expected to change more than semantics at this point, but some changes are expected on both).

That said, the `@staticInterop` approach is much much much closer to that future JSInteorp than any other pattern we support today (non-`@staticInterop` package:js, or the old and deprecated `dart:js`)

So my TLDR: migrating now will not save you from a migration later, but will let you experiment and experience something similar to the next-gen JSInterop